src/monstergenerator.cpp

```diff
@@ -431,6 +433,19 @@ void MonsterGenerator::load_monster(JsonObject &jo)
         newmon->fear = get_set_from_tags(fear_trig, trigger_map, MTRIG_NULL);
         newmon->placate = get_set_from_tags(placate_trig, trigger_map, MTRIG_NULL);
 
+        // Handle legacy flags
```

Comment:
It would probably better to completely remove those flags. Also remove the from the documentation in doc/JSON_FLAGS.md and please add documentation of the new json members to doc/JSON_INFO.md

Btw. vision_min is not really night vision, as it applies during day as well.